lib/players/human.rb

```diff
@@ -0,0 +1,7 @@
+class Human < Player
+
+  def move(token, board=nil)
+    input = gets.strip
+    return input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(token, board=nil)
  gets.strip
end
```